### PR TITLE
Napari episode

### PR DESCRIPTION
**Pull request by JeremyPike**

### CODE REVIEW

File: episodes/10-multidimensional-data.md [hunk 1]
```diff
@@ -0,0 +1,660 @@
...
+colour map. In Napari we can do this programmatically using the `channel_axis` variable:
+
+```python
+viewer.layers.clear()
+viewer.add_image(data=cells, name=["membrane", "cytoplasm", "nucleus"], channel_axis=0)
+```
+
+![](fig/cells-seperate-napari.png){alt='cells seperate napari ss'}
+
+Using what we have learnt in the previous Episodes lets segment the nuclei . First we produce a
```
Comment (JamesAllsopp) [inline episodes/10-multidimensional-data.md]: Pedantry alert; should that be let's?

File: episodes/10-multidimensional-data.md [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+binary mask of the nuclei by blurring the nuclei channel image and thresholding with the Otsu
+approach. Subsequently, we label the connected components within the mask to indentify individual
```
Comment (JamesAllsopp) [inline episodes/10-multidimensional-data.md]:
```suggestion
approach. Subsequently, we label the connected components within the mask to identify individual
```

File: episodes/10-multidimensional-data.md [hunk 3]
```diff
@@ -0,0 +1,660 @@
...
+bacteria colonies in an image also producing a new image that highlights the colonies. Modify this
+function to make use of Napari, as opposed to `Matplotlib`, to display both the original colony
+image
+and the segmented individual colonies. Test this new function on `"data/colonies-03.tif"`. If you
+did not complete the *Capstone challenge* epsidode you
+can start with function from the
+[solution](09-challenges.md#colony-counting-with-minimum-size-and-automated-threshold-optional-not-included-in-timing).
+Hints:
+
+1. A `napari.Viewer` object should be parsed to the function as an input parameter.
```
Comment (JamesAllsopp) [inline episodes/10-multidimensional-data.md]:
```suggestion
1. A `napari.Viewer` object should be passed to the function as an input parameter.
```

File: episodes/10-multidimensional-data.md [hunk 4]
```diff
@@ -0,0 +1,660 @@
...
+```
+
+![](fig/cells3d-slice-napari.png){alt='cell3d slice napari ss'}
+
+Note we now have a dimension slice to control which slice we are visualizing. You can also switch
+to a 3D rendering of the data:
+
+![](fig/cells3d-volume-napari.png){alt='cell3d volume napari ss'}
+
+Many of the scikit-image functions you have used throughout this Lesson work with 3D (or indeed nD)
```
Comment (JamesAllsopp) [inline episodes/10-multidimensional-data.md:360]: Not sure about the capital L for Lesson? Entirely up to you

File: episodes/10-multidimensional-data.md [hunk 5]
```diff
@@ -0,0 +1,660 @@
...
+print(sigma)
+```
+
+```output
+[1.72413793 1.92307692 1.92307692]
+```
+
+![](fig/cells3d-blurred-napari.png){alt='cell3d volume napari ss'}
+
+Note we have used a different sigma for each dimension which corresponds to 0.5 microns in real
```
Comment (JamesAllsopp) [inline episodes/10-multidimensional-data.md]:
```suggestion
Note we have used a different sigma for the z dimension to allow for the different voxel dimensions, to produce a blur of 0.5 microns in each axis of real space.
```

File: episodes/10-multidimensional-data.md (hunk 1 above)
Comment (JeremyPike) [inline episodes/10-multidimensional-data.md]:
```suggestion
Using what we have learnt in the previous Episodes let's segment the nuclei . First we produce a
```

File: episodes/10-multidimensional-data.md (hunk 4 above)
Comment (JeremyPike) [reply episodes/10-multidimensional-data.md:360]: Yes capital L for a Carpentries "Lesson" which is their word for the whole course

**Pull request merged**